app/helpers/GoogleData.js

```diff
@@ -4,67 +4,73 @@ import { LOCATION_DATA } from '../constants/storage';
  */
 import { GetStoreData, SetStoreData } from '../helpers/General';
 
-function BuildLocalFormat(placeVisit) {
-  return (loc = {
-    latitude: placeVisit.location.latitudeE7 * 10 ** -7,
-    longitude: placeVisit.location.longitudeE7 * 10 ** -7,
-    time: placeVisit.duration.startTimestampMs,
-  });
+/**
+ * Rounds float number to a desired number of decimal places and returns a float
+ * number. NOTE: .toFixed() returns a string, but number is required.
+ * @param num - number
+ * @param digits - amount of digits to round
+ * @returns {number}
+ */
+function toFixedNumber(num, digits) {
```

Comment:
[toPrecision](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Number/toPrecision) returns a string, but [toFixed](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed) returns a number.

Reply:
they actually both return "string", I think MDN also says the returning value is a string:

![Untitled 2](https://user-images.githubusercontent.com/1267571/79465636-9d27b280-7fc9-11ea-86ed-849946ba0087.png)

Reply:
What a trip! Chrome was a number for me. What browser is that? 

Or perhaps it's transpiled incorrectly...

Reply:
Bizarre... resolving this.